migration/i8.2z/oracle/202112181000_IDEMPIERE-5110.sql

```diff
@@ -0,0 +1,14 @@
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- Dic 18, 2021, 10:00:00 AM ART
+CREATE INDEX pp_product_bom_product_id_idx ON PP_Product_BOM (M_Product_ID)
+;
+
+-- Dic 18, 2021, 10:00:00 AM ART
+CREATE INDEX pp_product_bomline_pp_product_bom_id_idx ON PP_Product_BOMLine (PP_Product_BOM_ID)
```

Comment:
Please create the indexes in dictionary and then synchronize the index from there.

Alternatively, I think you can create the index in database as this script - but then please run the "Create Table Index" process to create the index on dictionary with official IDs

NOTE aside: this create index will not work in oracle 11 because of the name length, but we already decided that we are not supporting that anymore, so, not a problem, just a note.

Reply:
Done

Reply:
I must give you a developer ID to use here:
https://wiki.idempiere.org/en/Centralized_ID_Management

I'm in a meeting - will do it after